plugins/parsers/collectd/parser_test.go

```diff
@@ -221,7 +221,7 @@ func TestParse_SignSecurityLevel(t *testing.T) {
 
 	metrics, err = parser.Parse(bytes)
 	require.NoError(t, err)
-	require.Equal(t, []telegraf.Metric{}, metrics)
+	require.Equal(t, make([]telegraf.Metric, 0), metrics)
```

Comment:
Hmmm how about
```suggestion
	require.Empty(t, metrics)
```

Same for the instances below...

Reply:
Changed